.github/workflows/verify-pr-description.yml

```diff
@@ -12,7 +12,7 @@ permissions: read-all
 
 jobs:
   verify:
-    if: ${{ github.actor != 'dependabot[bot]' }}
+    if: ${{ !contains(github.event.pull_request.title, 'bump') }}
```

Comment:
**suggestion**: I'd add a comment to this line, so others would understand why we're doing this ;)